src/db.rs

```diff
@@ -1649,6 +1649,40 @@ impl WriteBatch {
         }
     }
 
+    pub fn delete_range<K: AsRef<[u8]>>(&mut self, start_key: K,  end_key: K) -> Result<(), Error> {
```

Comment:
Can you add some doc comments here and either a test or doc example so the CI build can validate this works?